plugins/plugin-git/che-plugin-git-ext-git/src/main/resources/org/eclipse/che/ide/ext/git/client/GitLocalizationConstant.properties

```diff
@@ -94,7 +94,7 @@ messages.delete_success=Git repository deleted
 messages.compare_save.title=Git compare 
 messages.compare_save.question=Would you like to save changes?
 messages.notAuthorizedTitle=You are not authorized to perform this operation
-messages.notAuthorizedContent=This may be because you have not setup SSH keys or oAuth.\
+messages.notAuthorizedContent=May be the authorization failed because you have not setup SSH keys or OAuth.\
```

Comment:
May I suggest `Authorization failure may be due to missing or incorrect SSH key or OAuth setup`